network-api/networkapi/mozfest/models.py

```diff
@@ -69,6 +69,7 @@ def get_template(self, request):
 
     def get_context(self, request, bypass_menu_buildstep=False):
         context = super().get_context(request)
+        context = set_main_site_nav_information(self, context, 'MozfestHomepage')
```

Comment:
A question just because I'm not familiar with Django / our existing setup -
Why do we need this now? Is it because the MozFest primary pages need to know about the `root` for the CTA button on nav to render properly?

Reply:
correct - all pages need to know where the homepage lives, so they can access its `cta_button_label` and `cta_button_destination` values